style.css

```diff
@@ -1,12 +1,20 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
 
+/*
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} */

```

Comment:
- [ ] While solving the issue that is addressed by the previous reviewer there are some unnecessary comments arise. Kindly avoid using unnecessary comments to make your codebase clean and readable.

Reply:
> - [ ] As requested by the previous code reviewer,  please ensure to remove unnecessary comments from your codebase. Including unnecessary comments can result in a cluttered, untidy, and less readable codebase, which could impact code maintainability and overall code quality.👍